lang/pt_BR.json

```
@@ -1,11 +1,169 @@
 {
+    ":Resource could not be changed to :value": ":Resource não pôde ser mudado para :value",
+    ":Resource could not be created": ":Resource não pôde ser criado",
+    ":Resource could not be deleted": ":Resource não pôde ser excluído",
+    ":Resource could not be restored": ":Resource não pôde ser restaurado",
+    ":Resource could not be updated": ":Resource não pôde ser atualizado",
+    ":Resource created successfully": ":Resource criado com sucesso",
+    ":Resource deleted successfully": ":Resource excluído com sucesso",
+    ":Resource restored successfully": ":Resource restaurado com sucesso",
+    ":Resource updated successfully to :value": ":Resource atualizado com sucesso para :value",
+    ":Resource updated successfully": ":Resource atualizado com sucesso",
+    ":firstItem - :lastItem of :total :resource": ":firstItem - :lastItem de :total :resource",
+    "About Me": "Sobre Mim",
+    "About": "Sobre",
+    "Active Players": "Jogadores Ativos",
+    "Activity": "Atividade",
+    "Appearance": "Aparência",
+    "Architect": "Arquiteto",
+    "Average Completion": "Média de Conclusão",
+    "Avg. Completion": "Média de Conclusão",
+    "Blue": "Azul",
+    "Color": "Cor",
+    "Community": "Comunidade",
+    "Connect": "Conecte-se",
+    "Country": "País",
+    "Currently Online": "Online no momento",
+    "Default": "Padrão",
+    "Dark": "Escuro",
+    "Date/Time Format": "Formato de Data/Hora",
+    "Developers Code of Conduct": "Código de Conduta dos Desenvolvedores",
+    "Display Preferences": "Preferências de Tela",
+    "Documentation": "Documentação",
     "Edit": "Editar",
-    "Save": "Salvar",
+    "Founder": "Fundador",
+    "Green": "Verde",
+    "Inbox": "Caixa de Entrada",
+    "Language": "Linguagem",
+    "Legal": "Legal",
+    "Light": "Claro",
+    "Message": "Mensagem",
+    "More": "Mais",
+    "New Achievements": "Novas Conquistas",
+    "No :resource": "Sem :resource",
+    "Number Format": "Formato de Número",
+    "Overview": "Visão Geral",
+    "Page :page of :total": "Página :page de :total",
+    "Pink": "Rosa",
+    "Play": "Jogar",
+    "Privacy Policy": "Políticas de Privacidade",
+    "Privacy": "Privacidade",
+    "Rank": "Classificação",
+    "Red": "Vermelho",
+    "Register": "Registrar",
+    "Save Settings": "Salvar Configurações",
+    "Search": "Pesquisar",
+    "Sign In": "Entrar",
+    "Sign Out": "Sair",
```

```suggestion
    "Sign In": "Fazer Login",
    "Sign Out": "Fazer Logout",
```
While "Entrar" and "Sair" are not wrong and would be understood, I suspect "Fazer Login" and "Fazer Logout" would be more familiar to Portuguese-speaking users when it comes to the context of accessing an account on a website or an application.

---

I used this words because all the big social midias that have portuguese translations use "Entrar" and "Sair" for Login and Logout and because of the simplicity of the words.

---

I've looked at a few websites, "Entrar" and "Sair" seem to be way more common. They feel more natural to the general public as they're shorter and don't involve English words.

---

I will definitely defer to you on this.